tools/Utilities/RegPreviewUtility/DevHome.RegistryPreview.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(SolutionDir)ToolingVersions.props" />
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+    <RootNamespace>DevHome.RegistryPreview</RootNamespace>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <PublishProfile Condition="'$(BuildingInsideVisualStudio)' != 'True'">Properties\PublishProfiles\win-$(Platform).pubxml</PublishProfile>
+    <UseWinUI>true</UseWinUI>
+    <EnableMsixTooling>false</EnableMsixTooling>
+    <ProjectPriFileName>DevHome.RegistryPreview.pri</ProjectPriFileName>
+    <EnableDefaultApplicationDefinition>false</EnableDefaultApplicationDefinition>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Page Remove="RegistryPreviewApp.xaml" />
+    <ApplicationDefinition Include="RegistryPreviewApp.xaml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240311000" />

```

Comment:
Similar comment to above about do we get this from DevHome.Common

Reply:
The big question for me is what happens when these versions get out of sync with the one specified in original build producing the nuget package. e.g. DevHome needs to update to 1.6 or 1.5.240312000. 

Does the power toys nuget package payload all need to be rebuilt with that updated reference as well? My understanding is that they must match exactly in order to build/deploy properly. So we need to make sure we have a path forward in the case that DevHome needs to update, but for whatever reason the power toys can't, what contingency plan do we have for that?

Even more practical, what's the local developer workflow just for testing a WinAppSDK version update to DevHome once this dependency is established?

Reply:
Yes, PowerToys nugets and DevHome needs to be in sync. I believe historically, PowerToys have updated their nugets first. In that scenario, DevHome can keep consuming older nugets (unless there's a breaking bug). 

For Dev inner loop, I will update the contributors guide to mention this and add notes on building PowerToys nugets locally.

Reply:
WASDK is back compatible and I think it should be fine if we want to update DevHome's WASDK before PowerToys'. I tried this locally and seems to work fine.